lib/event/eventFields.js

```diff
@@ -73,6 +74,18 @@ export const EventType = new GraphQLObjectType({
       type: DateTimeType,
       resolve: mapDateTime('endDateTime'),
     },
+    ticketReleaseDate: {
+      type: DateTimeType,
+      resolve: mapDateTime('ticketReleaseDate'),
+    },
+    ticketsAvailiable: {
+      type: GraphQLBoolean,
+      description: 'A boolean to show whether tickets are availiable',
+    },
+    eventActionLink: {
```

Comment:
i feel like there's potentially some duplication going on with the `externalLinks` and `internalLinks`. One option I think would be to extend `RelatedLinkType` to be featured as a "main" event action link.